tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/Testing/MsTestV2/TestContextStruct.cs

```diff
@@ -2,6 +2,7 @@
 // Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
 // This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
 // </copyright>
+#nullable enable
```

Comment:
Based on our previous discussion, I think `TestMethodContextStruct` should be `TestMethodContextStruct?` here

Reply:
true, done in [3d98a45](https://github.com/DataDog/dd-trace-dotnet/pull/6374/commits/3d98a45a5bebdc0c7de9990b062d3e8eb506713d)